src/game/shell.c

```diff
@@ -187,6 +190,100 @@ void Shell_Wait(int nticks)
     }
 }
 
+void Shell_ValidateLevelTitle(char *str)
+{
+    // Strip non alphanumeric chars
+    char *ix = str;
+    int idx = 0;
+    while (*ix != '\0') {
+        if ((*ix >= 'A' && *ix <= 'Z') || (*ix >= 'a' && *ix <= 'z')
+            || (*ix >= '0' && *ix <= '9') || *ix == '\'' || *ix == '.'
+            || *ix == ' ') {
+            idx++;
+            *ix++;
+        } else {
+            memmove(&str[idx], &str[idx + 1], strlen(str) - idx);
+        }
```

Comment:
I am not sure about this. I originally meant the sanitization to only apply to filenames, whereas this now covers also the presentation ingame. I know for a fact that the French community uses T1M to display text in French, and this uses non-alphanumeric characters outside the ASCII range. Let's do this elsewhere, in `Shell_GetScreenshotName`.